plugins/InstructionInspector/Plugin.cpp

```diff
@@ -1511,7 +1511,7 @@ InstructionDialog::InstructionDialog(QWidget *parent, Qt::WindowFlags f)
 			// TODO: mem_barrier
 			add({"Operands"});
 
-			auto *const operands = tree->topLevelItem(tree->topLevelItemCount() - 1);
+            auto *const operands = tree_->topLevelItem(tree_->topLevelItemCount() - 1);
```

Comment:
I don't quite get it: are we switching from tabs to spaces? BTW, the Style Guide has no mention of them.

Reply:
That would be an auto-format error. Sometimes clang-format gets confused and things it's aligning with spaes in certain contexts. fix this one up

Reply:
Actually, it is tabs in the latest update to the PR.